planetscale/branches.go

```diff
@@ -28,6 +29,7 @@ type databaseBranchesResponse struct {
 type CreateDatabaseBranchRequest struct {
 	Organization string `json:"-"`
 	Database     string `json:"-"`
+	Region       string `json:"region,omitempty"`
```

Comment:
We use `omitempty` here because the API will by default create a database in `us-east` if this is not sent over.